docs/reference/ingest/enrich.asciidoc

```diff
@@ -249,7 +249,10 @@ enrich policy executor.
 The enrich coordinator supports the following node settings:
 
 `enrich.cache_size`::
-Maximum number of searches to cache for enriching documents. Defaults to `1000`.
+Maximum number of searches to cache for enriching documents.
+By default, instead of looking at the _number_ of searches,
+the enrich coordinator looks at the size of the search hits in bytes and
+aims to only take up to 1% of the node's max heap space.
```

Comment:
Do we think this explanation is sufficient?  Or should I clarify `aims` - as in, why it's only an approximation?

Reply:
I think a small blurb about why we use "aims" here instead of "is" would be good